05-UI-ReactJS/react-builder-master/source/MovieInput.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+class MovieInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {title: "",year: "",duration: "", favourite: false};
+    this.handleChangeTitle = this.handleChangeTitle.bind(this);
```

Comment:
there's no need to assign the methods on the constructor, you have them assigned outside the method